xml/System.Security.Cryptography.Xml/EncryptedXml.xml

```diff
@@ -909,7 +909,6 @@
         <Parameter Name="symAlg" Type="System.Security.Cryptography.SymmetricAlgorithm" Index="1" FrameworkAlternate="xamarinmac-3.0" />
       </Parameters>
       <Docs>
-        <param name="plaintext">The data to encrypt.</param>
```

Comment:
this change seems wrong @joelmartinez. line 907 seems to indicate this is the parameter name for xamarin mac 3.0